### PR TITLE
Update the modal to work with new openseadragon

### PR DESCRIPTION
**Pull request by jonhartzler**
Why: the old modal was being used for the zoom link.  Also it was not working.  Now it works and we are using material-ui.  I also updated open seadragon

### CODE REVIEW

File: app/assets/javascripts/components/ItemImageZoomButton.jsx [hunk 1]
```diff
@@ -18,22 +26,44 @@ var ItemImageZoomButton = React.createClass({
...
+              label={ buttonLabel }
+              style={ buttonStyle }
+            />
+          </div>
+          <Dialog
+            ref="imageZoom"
+            title={this.props.modalTitle}
+            actions={this.okDismiss()}
+            openImmediately={false}
+            onDismiss={this.closeCallback}
```
Comment (jgondron) [inline app/assets/javascripts/components/ItemImageZoomButton.jsx]: Is closeCallback defined somewhere?

File: app/assets/javascripts/components/ItemImageZoomButton.jsx [hunk 2]
```diff
@@ -18,22 +26,44 @@ var ItemImageZoomButton = React.createClass({
...
+          <div className="hc-image-zoom-button-outer">
+            <FlatButton
+              onTouchTap={this.showModal}
+              label={ buttonLabel }
+              style={ buttonStyle }
+            />
+          </div>
+          <Dialog
+            ref="imageZoom"
+            title={this.props.modalTitle}
```
Comment (jgondron) [inline app/assets/javascripts/components/ItemImageZoomButton.jsx]: Also don't see this defined, or being passed into the component

**Comment by jgondron** [comment]
:+1:

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**